index.html

```diff
@@ -1,6 +1,31 @@
+<!-- index.html -->
+
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta http-equiv="refresh" content="0; url=docs/index.html" />
-  </head>
+<html>
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="//unpkg.com/docsify/themes/vue.css">
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css" title="vue">
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/dark.css" title="dark" disabled>
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/buble.css" title="buble" disabled>
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/pure.css" title="pure" disabled>
+</head>
+<body>
+  <div id="app"></div>
+  <script>
+    window.$docsify = {
+      loadSidebar: true,
+      auto2top: true,
+      coverpage: false,
+      name: "kubevirt",
+      repo: 'davidvossel/kubevirt-user-guide-v2',
```

Comment:
It should point to the kubevirt/user-guide repo instead, isn't it?

Reply:
Additionally ".travis.yml" should be removed as travis will not need to push any doc

Reply:
ha, nice catch thanks. I'll update the pr

Reply:
No worries, just in case, I did a mockup using mkdocs as part of the issue referenced in the description at https://kubevirt-user-guide-mockup.readthedocs.io/en/material/README.html

Thanks for the effort of updating all asciidoc to markdown files